lib/airbrake-ruby/async_sender.rb

```diff
@@ -24,7 +24,12 @@ def initialize(config)
     #   library
     # @return [nil]
     def send(notice)
-      @unsent << notice
+      if @unsent.size >= @unsent.max
+        msg = "#{LOG_LABEL} dropping notice since sending queue is full"
+        @config.logger.error msg
```

Comment:
I like the idea to log, so the information it's not lost and can be retrieved if necessary.

Having the queue full of errors and starting to not send them to the error log platform it's, for me, a critical situation. If, for some reason, I'm not aware of this situation and I'm looking at the logs I would like to see it quickly since it's an ERROR. What do you think ? 

I'm almost done with the changes, I'm having some trouble with the specs but I hope to fix it soon.